reconcile/osd_mirrors_data_updater.py

```diff
@@ -130,16 +130,21 @@ def run(dry_run, gitlab_project_id=None):
     # install-config.yaml compatible format
     mirrors_info = []
     for tfr in osd_mirrors:
-        image_url = get_image_uri(
-            aws_cli=aws_cli,
-            account=tfr['account'],
-            repository=tfr['identifier']
-        )
+        # ecr image is not in use for now
+        # image_url = get_image_uri(
+        #     aws_cli=aws_cli,
+        #     account=tfr['account'],
+        #     repository=tfr['identifier']
+        # )
+        quayio_url = tfr['mirror']['url']
+        ocmquay_url = quayio_url.\
+            replace('quay.io', 'pull.q1w2.quay.rhcloud.com')
```

Comment:
This integration goes through the ECR resources and get their quay.io mirrors.
I'm afraid that just changing the URL is too fragile. A new org in quay.io won't make this mirror include that org in the list.

Reply:
I mean, since ECR is out of the game now, we should completly skip that aws_cli logic.